src/openrct2/config/Config.cpp

```diff
@@ -659,6 +660,7 @@ namespace Config
         log_verbose("config_find_rct2_path(...)");
 
         static constexpr const utf8* searchLocations[] = {
+            R"(/usr/share/openrct2)",
```

Comment:
This one is a bit odd. While I somewhat understand the problem you're trying to solve, I'm not convinced this is the correct way. These paths are locations for known default places where RCT2 would be installed. `/usr/share/openrct2` is not one such location.

There should be a more platform-specific function already you should be able to use and then keep in mind the distinction between `OpenRCT2`, this project, and RCT2, the original game we need for its assets.

Reply:
I am open for suggestions :-)
I agree, that my path is not an original installation path.
Just didn't found a better way to patch that.

Where should I instead add the path?
Sounds like you have something in mind?

Reply:
Looking further into the code, I don't see any other places, where this could be definied. This seems to be exactly the place, where it just works, as intended.

I just updated for two paths, since you are right, that RCT1 and RCT2 data is not the same. 
But the functions seems to be exactly to do, in order to achieve that?

When I place the needed folders from RCT1 and RCT2 in both directories, applied with my patch, OpenRCT2 seems to work perfectly fine. It detects the stuff correctly and paths for `config.ini` are mapped correctly automatically.

About the subfolders, the names can be of course something different, maybe more `rct1_data` and `rct2_data` or something.

Reply:
@janisozaur @Gymnasiast ping